rideshare.rb

```diff
@@ -0,0 +1,138 @@
+# a method to add a driver to the driver hash
+def add_driver_to_hash(ride_hash, drivers_hash)
+  drivers_hash[ride_hash[:driver_id]] = {
```

Comment:
So any driver you add this this won't have any rides listed, but will have a cost and total of ratings?